mlflow/models/utils.py

```diff
@@ -41,9 +42,20 @@
     pd.DataFrame, np.ndarray, dict, list, "csr_matrix", "csc_matrix", str, bytes, tuple
 ]
 
-# TODO: update this
 PyFuncInput = Union[
-    pd.DataFrame, pd.Series, np.ndarray, "csc_matrix", "csr_matrix", List[Any], Dict[str, Any], str
+    pd.DataFrame,
+    pd.Series,
+    np.ndarray,
+    "csc_matrix",
+    "csr_matrix",
+    List[Any],
+    Dict[str, Any],
+    str,
+    bytes,
+    bool,
+    int,
+    float,
+    dt.datetime,
```

Comment:
```suggestion
    dt.datetime,
    np.ndarray,
    pd.DataFrame,
    pd.Series,
    "csc_matrix",
    "csr_matrix",
    Dict[str, Any],
    List[Any],
    bytes,
    bool,
    float,
    int,
    str,
```

minor nit: Group the same category together, and sort in each category.